### PR TITLE
login page and auth context

### PR DESCRIPTION
**Pull request by srishtigautam21**
added
Login page and auth context

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -7,12 +7,21 @@ import { formatDate } from "../utils/authUtils";
...
 export const users = [
+  // {
+  //   _id: uuid(),
+  //   firstName: "Adarsh",
+  //   lastName: "Balika",
+  //   email: "adarshbalika@gmail.com",
+  //   password: "adarshBalika123",
+  //   createdAt: formatDate(),
+  //   updatedAt: formatDate(),
+  // },
```
Comment (Maianki) [inline src/backend/db/users.js]: Remove commented code.

File: src/component/Navbar/navbar.css [hunk 2]
```diff
@@ -62,3 +62,23 @@
...
+}
+.menu-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.content-color {
+  color: var(--NOTIFICATION-COLOR);
+} */
```
Comment (Maianki) [inline src/component/Navbar/navbar.css]: remove commented code

**Review by Maianki — APPROVED** [review]
Everything looks good can remove commented code.

File: src/App.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import "./App.css";
-import { HomePage, VideoListingPage } from "./pages/index";
+import { HomePage, VideoListingPage, LoginPage } from "./pages/index";
```
Comment (dev-enforced) [inline src/App.js:2]: Without index also this import would work out!!

**Review by dev-enforced — APPROVED** [review]
The live preview is working good!! Couldn't navigate to the login and signup routes of live preview so do add a _redirects file in the public folder and add a button in the home page to navigate to the Login section!!
Code is also very well structured and easy to understand
Got introduced to a new custom Hook which you developed will be checking it i.e. useCustomAxios
Good work @srishtigautam21 !!

File: src/App.js (hunk 3 above)
Comment (srishtigautam21) [reply src/App.js:2]: thanks i am working on that button to navigate to login page

**Pull request merged**